### PR TITLE
ebs br: print out the elapsed time for each snapshot size calculation

### LINKED ISSUE #5264: ebs br: record the elapsed time for snapshot size calculation in backup pod log
## Feature Request

**Is your feature request related to a problem? Please describe:**

**Describe the feature you'd like:**

We need to record the elapsed time for snapshot calculation in log.

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by BornChanger**
### What problem does this PR solve?

Closes #5264 

### What is changed and how does it work?

This pr will directly print the elapsed time of snapshot size calculation in backup pod log.

### Code changes

- [X] Has Go code change
- [ ] Has CI related scripts change

### Tests

- [ ] Unit test 
- [ ] E2E test 
- [X] Manual test 
- [ ] No code 

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [ ] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note

```

### CODE REVIEW

File: cmd/backup-manager/app/util/backup_size.go [hunk 1]
```diff
@@ -199,6 +199,12 @@ func calcBackupSize(ctx context.Context, volumes map[string]string, level string
 func calculateSnapshotSize(volumeId, snapshotId string) (uint64, uint64, error) {
 	var snapshotSize uint64
 	var numApiReq uint64
+
+	start := time.Now()
+
+	klog.Infof("start to calculate snapshot size for %s, base on snapshot %s, volume id %s",
```
Comment (WangLe1321) [inline cmd/backup-manager/app/util/backup_size.go:205]: the number of placeholder is more than the number of values.

Comment (BornChanger) [reply cmd/backup-manager/app/util/backup_size.go]: fixed.

**Review by WangLe1321 — APPROVED** [review]

**Pull request merged**